roles/osp_observability/templates/prometheus.yml.j2

```diff
@@ -2,17 +2,34 @@
 global:
   scrape_interval: {{ prometheus_scrape_interval }}
   external_labels:
-    {% if ansible_fqdn %}agent_host: {{ ansible_fqdn }}{% endif %}
-    {% if openstack_cloud_name %}cloud_name: {{ openstack_cloud_name }}{% endif %}
+    {% if ansible_fqdn -%}
```

Comment:
Didn't review below this because it seems to match the other review. Let me know if something is different here that needs a look.

Reply:
It doesn't QUITE match the other review because that one didn't pass the new yamllint testing. I think the only change was outdenting any tags that appear before empty lines to avoid trailing-whitespace errors.